activestorage/lib/active_storage/analyzer/image_analyzer/image_magick.rb

```diff
@@ -1,22 +1,17 @@
 # frozen_string_literal: true
 
+begin
+  gem "mini_magick"
+rescue LoadError => error
+  raise error unless error.message.include?("ruby-vips")
```

Comment:
Copy-paste error? I think this will cause it to not show the deprecation warning.
```suggestion
  raise error unless error.message.include?("mini_magick")
```

Reply:
https://github.com/rails/rails/commit/2641be96e16b1263b6e673664fbb254844c1ceee

Reply:
:facepalm: Thanks, was fixing a rubocop failure